test/jdk/javax/swing/JRadioButton/8033699/bug8033699.java

```diff
@@ -68,6 +68,10 @@ public static void main(String[] args) throws Throwable {
 
     private static void testLaF(UIManager.LookAndFeelInfo laf) throws Exception {
         try {
+            SwingUtilities.invokeAndWait(() -> {
+                focusManager = KeyboardFocusManager.getCurrentKeyboardFocusManager();
+            });
+
```

Comment:
Storing the `focusManager` once before calling `testLaF` for the first time is enough.